src/components/event-listing/EventListing.tsx

```diff
@@ -8,6 +8,8 @@ import { DEFAULT_IMAGE } from '../../utils/copy';
 
 const { Title, Text, Paragraph } = Typography;
 
+const BASE_EVENTS_ROUTE = '/events/';
```

Comment:
❓ I wonder if we can pull from the Routes export from App.tsx, although that may not have the bare /events/ route... just a thought! I might be wrong here